### PR TITLE
10 change namespace url to be resolvable

### PR DESCRIPTION
**Pull request by SArndt-TIB**
Adresses #10

### CODE REVIEW

**Review by andrecastro0o — COMMENTED** [review]
i have added some questions

File: w3id/.htaccess [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+# HTML:
+# RewriteCond %{HTTP_ACCEPT} text/html
+# RewriteRule ^$ https://nfdi4ing.pages.rwth-aachen.de/metadata4ing/metadata4ing/index.html [R=303,L,NE]
+
+RewriteCond %{HTTP_ACCEPT} text/html
+RewriteRule ^(2020)$ https://terminology.tib.eu/ts/ontologies/dfgfo [R=303,L,NE]
+
+RewriteCond %{HTTP_ACCEPT} text/html
+RewriteRule ^([2-9]\d{3})$ https://terminology.tib.eu/ts/ontologies/dfgfo$1 [R=303,L,NE]
```
Comment (StroemPhi) [inline w3id/.htaccess:15]: As long as we don't have versions of an ontology stored in the TS we should resolve to the ontology PURL here as well.

Comment (andrecastro0o) [reply w3id/.htaccess:15]: thanks @StroemPhi

**Comment by SArndt-TIB** [comment]
@andrecastro0o I removed the .htaccess file from this repo and just left the w3id folder with a readme.md and a reference to the w3id repo on github and our folder there. Since Doro asked for reachability of all versions, i would ingest 2024 in addition to 2020.

**Comment by SArndt-TIB** [comment]
@andrecastro0o I reolved all conflicts that were introduced by merging #33  Should be okay now, any changes to the w3id would need to be done in the other repo. @doigl already had her PR pulled for removing the dots.

**Review by andrecastro0o — APPROVED** [review]
looks good. feel fee to merge

**Pull request merged**